### ISSUE TITLE
Feature Request: Extending support to material3

### ISSUE THREAD

**Issue by DhanashreeBhaskar09** [cashapp/paparazzi#573]
In our current project we are using compose material. However, with the introduction of material3 we have started to use it too.

Paparazzi snapshots with material work smoothly but it seems to be having issues when added material3 composables. In our case **all** snapshot tests with / without material3 composable errors out with below error message - 
java.lang.AssertionError: Images differ (by `some decimal number`%) 
When tests are run locally all the images generated are blacked out.

It would be great if we have paparazzi extending support to material3 composable.

**Comment by wezley98** [comment]
```
app.cash.paparazzi.internal.PaparazziLogger$MultipleFailuresException: There were 4 errors:
  
    java.lang.NoSuchMethodError: 'android.text.BoringLayout$Metrics android.text.BoringLayout.isBoring(java.lang.CharSequence, android.text.TextPaint, android.text.TextDirectionHeuristic, boolean, android.text.BoringLayout$Metrics)'
  	at androidx.compose.ui.text.android.BoringLayoutFactory33.isBoring(BoringLayoutFactory.kt:141)
  	at androidx.compose.ui.text.android.BoringLayoutFactory.measure(BoringLayoutFactory.kt:50)
  	at androidx.compose.ui.text.android.LayoutIntrinsics$boringMetrics$2.invoke(LayoutIntrinsics.kt:46)
  	at androidx.compose.ui.text.android.LayoutIntrinsics$boringMetrics$2.invoke(LayoutIntrinsics.kt:44)
  	at kotlin.UnsafeLazyImpl.getValue(Lazy.kt:81)
```

**Comment by JPUF** [comment]
I also found that having a material3 dependency caused the screenshot tests to fail & generate a 'blacked out'/blank screenshot. 

```
app.cash.paparazzi.internal.PaparazziLogger$MultipleFailuresException: There were 2 errors:

  java.lang.NullPointerException: null
	at androidx.core.os.BuildCompat.isAtLeastPreReleaseCodename(BuildCompat.java:60)
	at androidx.core.os.BuildCompat.isAtLeastT(BuildCompat.java:211)
	at androidx.compose.ui.text.android.BoringLayoutFactory.measure(BoringLayoutFactory.kt:49)
	...
	
java.lang.IllegalStateException: layout state is not idle before measure starts
	at androidx.compose.ui.node.LayoutNodeLayoutDelegate.performMeasure-BRTryo0(LayoutNodeLayoutDelegate.kt:1068)
```

Whereas I had no issue with only a `material` dependency  (not m3).

**Comment by jamesbApadmi** [comment]
Yep! The temporary solution posted by saintmarina in #631  works for me. Do you know what release will #640 be included in?

**Comment by jrodbx** [comment]
Closed by #640

**Comment by swaroopreddymukku** [comment]
Hi this is still happening for me, I'm using Paparazzi 1.3.4 which is supposed to have the fix but several snapshots are getting blanked out
